js/packages/common/src/contexts/connection.tsx

```diff
@@ -365,14 +376,16 @@ export const sendTransactions = async (
 
 export const sendTransaction = async (
   connection: Connection,
-  wallet: any,
+  wallet: WalletSigner,
```

Comment:
`WalletSigner` is just `{pubkey, signTransaction, signAllTransactions}` from the context, the minimal interface for signing transactions throughout the actions in the web package.